src/pages/Home/index.tsx

```diff
@@ -52,19 +59,25 @@ const Homepage = ({ order, setOrder }: OrderProps): JSX.Element => {
   };
 
   useEffect(() => {
+    if(data){
+      setFilteredBobaList(data)
+    }
     if (searchQuery) {
-      const filterlist = filterBobaList(bobaList, searchQuery);
+      const filterlist = filterBobaList(data, searchQuery);
       if (filterlist.length === 0) {
```

Comment:
Can you revisit this logic?
if the array is zero it's empty array, why can't you just set it directly?
```setFilteredBobaList(filterlist)```

Reply:
updated on 57d15d43a8edae133c80140c5cf2abb035c6aaf0